config/locales/sv.yml

```diff
@@ -214,18 +214,26 @@ sv:
       success: Din ansökan har lämnats in för en annan granskning.
       error: Det uppstod ett fel när du försöker att skicka in din ansökan för en annan granskning.
 
-    new_status: &status_new Nytt
+    state:
+      new: &status_new Nytt
```

Comment:
Whether its "Ny" or "Nytt" depends on what specifically it is that is new. 
Ansökan ends with "n" therefore wording should be "Ny"